routes/v1.js

```diff
@@ -46,99 +93,131 @@ api.declare({
   deferAuth:      true,
   scopes:         ['aws-provisioner:create-worker-type:<workerType>'],
   input:          SCHEMA_PREFIX_CONST + 'create-worker-type-request.json#',
-  output:         SCHEMA_PREFIX_CONST + 'create-worker-type-response.json#',
+  output:         SCHEMA_PREFIX_CONST + 'get-worker-type-response.json#',
   title:          "Create new Worker Type",
   description: [
-    // Document what this method does
+    "Create a worker type and ensure that all EC2 regions have the required ",
+    "KeyPair"
   ].join('\n')
-}, function(req, res) {
+}, 
+function(req, res) {
   var ctx         = this;
   var input       = req.body;
   var workerType  = req.params.workerType;
 
-  // Authenticate request by providing parameters, this is necessary because
-  // we set `deferAuth: true`, we can't do automatic authentication if the
-  // scopes contain parameters like <workerType>
   if(!req.satisfies({
     workerType:       workerType
   })) {
-    return; // by default req.satisfies() sends a response, so we're done
+    return; // by default req.satisfies() sends a response on failure, so we're done
   }
 
-  // TODO: If workerType definition specifies scopes that should be given
+  // TODO: If workerType launchSpecification specifies scopes that should be given
   //       to the workers using temporary credentials, then you should validate
   //       that the caller has this scopes to avoid scope elevation.
+  // TODO: ^ do this.  not entirely sure what this means, are LaunchSpecification
+  //       security groups the same as scopes?
```

Comment:
Nope....

We should add a list of scopes that should given in temporary credentials to workers created with the launchSpecification. When we do this, we should require this scopes from the person creating the workerType to avoid scope elevation issues.